documentation/book/proc-configuring-kafka-mirror-maker-authentication-scram-sha-512.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='proc-configuring-kafka-mirror-maker-authentication-scram-sha-512-{context}']
+= Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+* Username of the user which should be used for authentication
```

Comment:
Username ... user ... used!  Just "Username to be used for authentication" would do, I think.